keydb/templates/sa.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.serviceAccount.enabled | and .Values.serviceAccount.create }}
```

Comment:
Please explain how `| and` construction works.

Reply:
This is just a logical AND https://helm.sh/docs/chart_template_guide/function_list/#and
formatted as a pipeline (the expression is equivalent to `and .Values.serviceAccount.create .Values.serviceAccount.enabled`).

Reply:
Got it. Just forget how ugly gotpl could be :)